fe/fe-core/src/main/java/com/starrocks/analysis/CreateMaterializedViewStmt.java

```diff
@@ -198,6 +198,11 @@ public void analyzeSelectClause(boolean ignoreCast) throws AnalysisException {
                 FunctionCallExpr functionCallExpr = (FunctionCallExpr) selectListItemExpr;
                 String functionName = functionCallExpr.getFnName().getFunction();
                 MVColumnPattern mvColumnPattern = FN_NAME_TO_PATTERN.get(functionName.toLowerCase());
+                // current version not support count(distinct) function in creating materialized view
+                if (functionName.toLowerCase().equals(FunctionSet.COUNT) && functionCallExpr.isDistinct()){
```

Comment:
Better not put it in the middle of mvColumnPattern check

Reply:
Use ignorecaseEquals

Reply:
And, is `sum(distinct)` supported? If it is not supported, why not disable count(distinct) here.

Reply:
I agree, maybe adding a MVColumnCountPattern is a better way ?